AdvancedTaskSpecFlow/AdvancedTaskSpecFlow/Pages/EditShareSkillPage.cs

```diff
@@ -0,0 +1,236 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using AutoIt;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestPlatform.Utilities;
+using MongoDB.Bson;
+using AdvancedTaskSpecFlow.Models;
+
+namespace AdvancedTask.Pages
+{
+    public class EditShareSkillPage
+    {
+        readonly IWebDriver driver;
+        public EditShareSkillPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+        private IWebElement Title => driver.FindElement(By.XPath("//*[@id='service-listing-section']/div[2]/div/form/div[1]/div/div[2]/div/div[1]/input"));
+        private IWebElement Description => driver.FindElement(By.XPath("//*[@id='service-listing-section']/div[2]/div/form/div[2]/div/div[2]/div[1]/textarea"));
+        private IWebElement CategoryDropDown => driver.FindElement(By.XPath("//*[@id='service-listing-section']/div[2]/div/form/div[3]/div[2]/div/div[1]/select"));
+        private IWebElement SubCategoryDropDown => driver.FindElement(By.XPath("//*[@id='service-listing-section']/div[2]/div/form/div[3]/div[2]/div/div[2]/div[1]/select"));
+        private IWebElement Tags => driver.FindElement(By.ClassName("ReactTags__tagInputField"));
+        private IWebElement StartDateDropDown => driver.FindElement(By.XPath("//*[@id='service-listing-section']/div[2]/div/form/div[7]/div[2]/div/div[1]/div[2]/input"));
+        private IWebElement EndDateDropDown => driver.FindElement(By.XPath("//*[@id='service-listing-section']/div[2]/div/form/div[7]/div[2]/div/div[1]/div[4]/input"));
+        private IWebElement SkillExchange => driver.FindElement(By.XPath("//div[@class='form-wrapper']//input[@placeholder='Add new tag']"));
+        private IWebElement CreditAmount => driver.FindElement(By.Name("charge"));
+        private IWebElement Save => driver.FindElement(By.XPath("//input[@value='Save']"));
+        private IWebElement CancelButton => driver.FindElement(By.XPath("//*[@id=\"service-listing-section\"]/div[2]/div/form/div[11]/div/input[2]"));
+
+        public void editShareSkill(string title, string description, string category, string subCategory, List<string> tagsToRemove, List<string> tagsToAdd, string serviceType, string locationType, AvailableDaysModel availableDays, string skillTrade, string credit, List<String> skillExchangeTagsToRemove, List<String> skillExchangeTagsToAdd, string workSampleFilename, string active)
```

Comment:
can you please break this method down to smaller methods

Reply:
Hi Aman, 

I've broken the editShareSkill method into smaller methods which should help with readability.